src/components/Home.tsx

```diff
@@ -5,13 +5,18 @@ import Header from './Header'
 import Content from './Content'
 import { AvatarList, Spinner, Whiteboard } from './Whiteboard'
 import type { Shape, User } from '../types'
-import { SessionBackendProvider, useEventListener, useReady, useSend } from '@jamsocket/javascript/react'
+import { SessionBackendProvider, useReady } from '@jamsocket/javascript/react'
+import {SocketIOProvider, useEventListener, useSend} from "@jamsocket/javascript/socketio"
```

Comment:
nit: lets put spaces between the `{}` and the names inside so it looks exactly like the README code examples